src/lib/components/identity/identity-details.svelte

```diff
@@ -16,6 +16,7 @@
     export let actionButtons: ActionButton[] = []
 
     let revoking: boolean = false
+    let caret: boolean = true
```

Comment:
I think caret doesn't has to be its own variable. Why not just using `!loading` as caret?